showcase/src/app/demo/footer/footer.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Inject, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+@Component({
+	selector: "showcase-demo-footer",
+	templateUrl: "./footer.component.html"
+})
+export class FooterComponent implements OnInit {
```

Comment:
The name of this component should be `DemoFooterComponent` to avoid confusions with the actual component from Stark UI.